README.md

```
@@ -33,14 +33,11 @@ To configure additional external config file for Spring Boot application set up
 
 #### Config file
 
-There are five config files:
-- `application.yml` - default config file
-- `application-dev.yml` - config file for DEV environment
-- `application-test.yml` - config file for TEST environment
-- `application-prod.yml` - config file for PROD environment
+There are two config files in the repository:
+- `application.yml` - default config file for local run
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)